src/main/java/jarvis/Jarvis.java

```diff
@@ -0,0 +1,90 @@
+package jarvis;
+
+import java.util.HashMap;
+
+import enums.Command;
+import exception.JarvisException;
+import parser.Parser;
+import storage.Storage;
+import task.TaskList;
+import ui.Ui;
+
+public class Jarvis {
+    private Ui ui;
+    private TaskList tasks;
+    private Storage storage;
+
+    private boolean processNext = true;
+
+    public Jarvis() {
+        startup();
+    }
+
+    /**
+     * Runs the main chat/program loop.
+     *
+     * @return The response given based on the input.
+     */
+    public String getResponse(String input) {
+        while (processNext) {
+            try {
```

Comment:
Should parsedCommand be renamed to parsedCommands since it is a HashMap, which is itself a collection?